scripts/setDefaultIndex.py

```diff
@@ -45,11 +47,22 @@ def add_missing_elements_to_dict(to_verify, original_content):
             missing_elements[key] = original_content[key]
     return missing_elements
 
+def flatten_dict(d):
```

Comment:
Add header comment with 2 different examples to show how/why this function is used